src/Directory.Packages.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <AkkaVersion>1.5.51</AkkaVersion>
-    <AkkaHostingVersion>1.5.51</AkkaHostingVersion>
+    <AkkaHostingVersion>1.5.51.1</AkkaHostingVersion>
```

Comment:
Had to upgrade to a newer version of Akka.Hosting to resolve the health check registration bug.